templates/default/puma.monitrc.erb

```diff
@@ -0,0 +1,11 @@
+<% pid_file = "/run/lock/#{@app_shortname}/#{@appserver_name}.pid" %>
+<% app_dir = File.join(@deploy_to, 'current') %>
+<% config_file = File.join(@deploy_to, 'shared/config/puma.rb') %>
+<% env = @environment.map {|k,v| "#{k}=\"#{v}\""}.join(' ') %>
+<% pumactl = "#{env} bin/pumactl -F #{config_file}"%>
```

Comment:
TODO: This assumes that a puma binstub has been checked into the codebase, which may not be true for all users.  Probably should be changed to `bundle exec pumactl`